src/CMakeLists.txt

```diff
@@ -104,7 +104,11 @@ if (NNG_PLATFORM_POSIX)
     )
 endif()
 
-if (NNG_HAVE_KQUEUE)
+if (NNG_HAVE_EPOLL)
```

Comment:
I actually think kqueue should have precedence.  Some systems are emulating epoll, and I won't be surprised if BSDs do that too to facilitate portability, but kqueue is definitely better.  epoll() itself is rather defective under the hood, and we would prefer not to use it unless no other alternative is viable.

Reply:
done in 7d8e166